tasks/kernel.yml

```diff
@@ -34,19 +34,22 @@
     state=present
   when: ansible_os_family == 'RedHat'
 
-## Haven't been tested yet.
-#- name: Add ulimit into /lib/lsb/init-functions
-#  lineinfile:
-#    dest=/lib/lsb/init-functions
-#    insertafter='^export PATH'
-#    line='ulimit -n 65536'
-#    state=present
-#  when: ansible_os_family == 'Debian'
+# see how to set default limit system-wide for systemd units
+# https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html
+# https://serverfault.com/questions/958760/changing-ulimit-on-systemd
+- name: 'default open file limit for systemd units'
+  lineinfile:
+    dest=/etc/systemd/system.conf
```

Comment:
Suggest to put this into `system.conf.d/*.conf`, as recommended by systemd.
https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#:~:text=Using%20drop%2Dins%20for%20local%20configuration%20is%20recommended%20over%20modifications%20to%20the%20main%20configuration%20file.

Reply:
thanks for the advice.

use `/etc/systemd/system.conf.d/systemd_openfile.conf` instead.